package.json

```diff
@@ -446,6 +446,7 @@
     "test-runner": "jest"
   },
   "react-native": {
+    "@tanstack/react-query": "@tanstack/react-query/build/esm/index",
```

Comment:
We want to reference the ESM build, and not the UMD build.

Reply:
Waiting on [this one?](https://github.com/TanStack/query/issues/3882) I'll subscribe to updates 🤓